src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitShopService {
+    boolean applySuitableOperation(List<FruitRecordDto> fruitRecordDtos);
+
+    Map<Fruit, Integer> fruitStorage();
```

Comment:
No need in this method as you can access db directly, what you are actually doing in your strategies

Reply:
done